andela/wg-leo
# Add Fitbit support - 158626058

**What does this PR do?**
Add Fitbit support

**Description of the completed task**
Add a page and a button that allows the user to authorize Wger on the Fitbit authorization page, Wger gets a token for accessing the user's data and uses it to get the user's data and stores it.

**How should this be manually tested?**
- Clone the repo, ```cd``` into the wg-leo directory and run ```./manage.py runserver```. 
- Navigate to the URL provided by the development server (usually ```localhost```) using a browser.
- Create an account or log in
- Navigate to ```Dashboard``` at the Navigation bar.
- Click ```Sync with Fitbit``` button at the bottom of the page.
- Click again on ```Sync with Fitbit``` button on the loaded page.
- Login to your Fitbit account.
- Check ```profile```, ```weight``` and ```activity and exercises``` check boxes.
- Click ```allow``` button to sync the checked data.
- Finally nagivate to weight overview to check your weight.


**What are relevant pivotal tracker stories?**
[#158626058](https://www.pivotaltracker.com/story/show/158626058)

**Screenshots**
<img width="1278" alt="screen shot 2018-07-11 at 09 33 36" src="https://user-images.githubusercontent.com/24792513/42554534-edba63d8-84ed-11e8-8aa6-6bdd1789e4f2.png">
<img width="1278" alt="screen shot 2018-07-11 at 09 33 53" src="https://user-images.githubusercontent.com/24792513/42554555-ff3bcdd6-84ed-11e8-9ec4-c84219034b01.png">
<img width="1275" alt="screen shot 2018-07-11 at 09 34 31" src="https://user-images.githubusercontent.com/24792513/42554569-0ce13584-84ee-11e8-9132-5554eb1cff9c.png">
<img width="1269" alt="screen shot 2018-07-11 at 09 35 11" src="https://user-images.githubusercontent.com/24792513/42554575-18179f60-84ee-11e8-8704-ddd86c7fd902.png">
<img width="1273" alt="screen shot 2018-07-11 at 09 49 27" src="https://user-images.githubusercontent.com/24792513/42555264-500131dc-84f0-11e8-9eee-72bb27aa308c.png">
<img width="1277" alt="screen shot 2018-07-11 at 09 49 11" src="https://user-images.githubusercontent.com/24792513/42555306-762d7ff0-84f0-11e8-9527-c76f4bed2e7c.png">
